### PR TITLE
feat: [PROD-10520] Add DataIngestionInProgress state

### PR DESCRIPTION
**Pull request by ardvi**

### CODE REVIEW

File: tests/samples/DashboardSample.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    label: 'ut_noScenario_label',
+  },
+  noDashboard: {
+    label: 'ut_noDashboard_label',
+  },
+  noRun: {
+    label: 'ut_noRun_label',
+  },
+  inProgress: {
+    label: 'ut_InProgress_label',
```
Comment (esasova) [inline tests/samples/DashboardSample.js:16]: just a question: all labels begin with a lowercase letter and this one begins with a capitalized _I,_ is it on purpose ? :slightly_smiling_face:

**Review by esasova — APPROVED** [review]
LGTM :+1:

File: src/charts/SimplePowerBIReportEmbed/SimplePowerBiReportEmbed.spec.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import { render } from '@testing-library/react';
+
+import { TypographyTesting } from '../../../tests/MuiComponentsTesting';
+
+import { SimplePowerBIReportEmbed } from '.';
+import { LABELS, DEFAULT_SCENARIO, SCENARIO_STATES } from '../../../tests/samples/DashboardSample';
+
+const defaultProps = {
+  index: 0,
+  refreshTimout: 15000,
```
Comment (csm-thu) [inline src/charts/SimplePowerBIReportEmbed/SimplePowerBiReportEmbed.spec.js:11]: Typo: `refreshTimout` -> `refreshTimeout`

**Review by csm-thu — APPROVED** [review]
One typo to fix, but otherwise LGTM :+1:

File: src/charts/SimplePowerBIReportEmbed/SimplePowerBiReportEmbed.spec.js (hunk 2 above)
Comment (ardvi) [reply src/charts/SimplePowerBIReportEmbed/SimplePowerBiReportEmbed.spec.js]: ok

**Pull request merged**